src/ontology/cl-edit.owl

```diff
@@ -31685,6 +31688,50 @@ AnnotationAssertion(rdfs:label obo:CL_4033056 "onychocyte")
 SubClassOf(obo:CL_4033056 obo:CL_0000312)
 SubClassOf(obo:CL_4033056 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0008198))
 
+# Class: obo:CL_4033057 (luminal adaptive secretory precursor cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19648928") Annotation(oboInOwl:hasDbXref "PMID:35617956") Annotation(oboInOwl:hasDbXref "https://nrs.harvard.edu/URN-3:HUL.INSTREPOS:37368328") Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-6677-8489") obo:IAO_0000115 obo:CL_4033057 "A luminal epithelial cell of the mammary gland that can proliferate and has the potential to differentiate into a lactocyte during pregnancy. In humans, a luminal adaptive secretory precursor cell can be identified by high levels of the markers EPCAM and CD49f, and in mice, it can be identified by low levels of CD29 and high levels of CD14, Kit, CD61, and TSPAN8.")
```

Comment:
- "lactocyte" is not defined in the ontology.
- "EPCAM" <-- should this be "EpCAM"?
- " ... and in mice, it can be... " <-- comma should be removed
- "TSPAN8" <-- should this be "Tspan8"?

Reply:
Corrected